### ISSUE TITLE
Possible to plot two different chromosomes? 

### ISSUE THREAD

**Issue by ahishsujay** [schneebergerlab/plotsr#14]
Hi Manish, 

I wanted to investigate certain regions that fall in two different chromosome regions between my query and reference genomes and was wondering if there's anyway to visualize this? I know it would only make sense to visualize (SYN, INV, DUP, etc.) regions on the same chromosome but was interested on your inputs if it's possible to do any sort of visualize between two different chromosomes. 

Thanks!
Ahish

**Comment by mnshgl0110** [comment]
Do you mean inter-chromosomal rearrangements? If so, then you can use the `--itx` option with the `--chr` options to view them (check example R9 [here](https://github.com/schneebergerlab/plotsr/blob/master/example/plotsr.sh)). However, this plots entire chromosomes as it is currently not possible to zoom in to two separate regions. Maybe, I will add this later.

**Comment by ahishsujay** [comment]
Yes, I wanted to visualize inter-chromosomal rearrangements and to zoom in to two separate regions as well. No problem as viewing just the inter-chromosomal rearrangements will suffice for now! 

Thanks!

**Comment by mnshgl0110** [comment]
I see. Good that the issue is resolved for now. I would leave this issue open though, so that I remember to add multi-region zooming later.

**Comment by JakobButler** [comment]
Hello Manish,
I thought I'd piggy back on this issue since it's related. I'm also interested in looking at inter-chromosomal rearrangements using --itx, but the two genomes I'm comparing have a lot of _intra_-chromosomal rearrangements (translocations and duplications) as well which makes the figures a bit hard to interpret. I attempted to modify my syri.out file and remove every feature with a matching chromosome name to clear these off the plots, but plotsr didn't like that much at all (File "/u/xxx/.conda/envs/syri/lib/python3.8/site-packages/plotsr/func.py", line 388, in readsyriout
    chrid.append((i, np.unique(df.loc[(df[0] == i) & (df[10] == 'SYN'), 5])[0]))
IndexError: index 0 is out of bounds for axis 0 with size 0).

Have you got any thoughts about what to change to show only inter-chromosomal features?

**Comment by mnshgl0110** [comment]
Your modified input file is missing SYN regions which are require to select homologous chromosomes. You can filter intra-chromosomal SRs and set colour for SYN regions to white in the base.cfg file. Probably, that would work.

**Comment by HLHsieh** [comment]
Hi Manish,

Thank you for developing such a useful tool. I’m currently using it to investigate certain genomic regions and analyze assembly results. Therefore, I wanted to follow up on this issue or approach. I would greatly appreciate it if you could provide any guidance or suggestions.

Best regards,
Hsin
